packages/main/src/themes/TableGroupRow.css

```diff
@@ -8,18 +8,20 @@
 }
 
 .ui5-table-group-row-root {
-	height: 2rem;
-	border-bottom: 1px solid var(--sapList_TableGroupHeaderBorderColor);
+	height: var(--ui5_table_group_row_height);
+	border-style: solid;
+	border-color: var(--sapList_TableGroupHeaderBorderColor);
+	border-width: var(--ui5-table_border_width);
```

Comment:
`--ui5_table_border_width`, with an underscore b/w ui5 and table, change the other 5 occurrences too.